### PR TITLE
Change annotations to attributes

### PR DESCRIPTION
**Pull request by aerrasti**
## Subject

Change annotations to attributes.

Follow instructions https://api-platform.com/docs/core/upgrade-guide/

## To dos

- [x] fix `changePassword` and `changePasswordLogged` 
- [x] convert to attributes `User` and `Fishbowl` entities

### CODE REVIEW

**Review by eddsaura — APPROVED** [review]

File: backend/packages/Core/src/Entity/User.php [hunk 1]
```diff
@@ -33,171 +39,132 @@
...
+        new GetCollection(controller: NotFoundAction::class, output: false, read: false),
+        new Post(
+            denormalizationContext: ['groups' => ['user:write', 'user:create']],
+            validationContext: ['groups' => ['Default', 'user:create']]
+        ),
+    ],
+    normalizationContext: ['groups' => ['user:read']],
+    denormalizationContext: ['groups' => ['user:write']],
+    graphQlOperations: [
+            new Query(
```
Comment (jordisala1991) [inline backend/packages/Core/src/Entity/User.php:55]: indentations here looks wrong.

File: backend/packages/Core/src/JWT/Model/JWTToken.php [hunk 2]
```diff
@@ -16,6 +16,7 @@
 use App\Core\JWT\Model\Payload\FeaturesPayload;
 use App\Core\JWT\Model\Payload\HeaderPayload;
 use App\Core\JWT\Model\Payload\UserPayload;
+use DateTimeImmutable;
```
Comment (jordisala1991) [inline backend/packages/Core/src/JWT/Model/JWTToken.php:19]: is this change related? we don't seem to use this kind of use on the code.

File: backend/packages/Fishbowl/src/Service/FishbowlService.php [hunk 3]
```diff
@@ -22,15 +22,15 @@
...
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\RequestStack;
-use Symfony\Component\Security\Core\Security;
 use Symfony\Component\Security\Core\User\UserInterface;
 use Symfony\Contracts\Translation\TranslatorInterface;
 use Webmozart\Assert\Assert;
 
 /**
- * @psalm-type RawParticipant = array{
+ * @psalm-type RawParticipant array{
```
Comment (jordisala1991) [inline backend/packages/Fishbowl/src/Service/FishbowlService.php:33]: Is this correct?

Comment (aerrasti) [reply backend/packages/Fishbowl/src/Service/FishbowlService.php:33]: thanks!
